docker/openemr/5.0.2/php.ini

```diff
@@ -1345,14 +1345,14 @@ session.save_handler = files
 ; applications from session fixation via session adoption vulnerability. It is
 ; disabled by default for maximum compatibility, but enabling it is encouraged.
 ; https://wiki.php.net/rfc/strict_sessions
-session.use_strict_mode = 0
+session.use_strict_mode = 1
```

Comment:
@bradymiller Is this permissible? Has OpenEMR been properly tested for compatibility with this flag?

Reply:
hi @jesdynf , I need to do some research on this. -brady